DESCRIPTION

```diff
@@ -8,7 +8,7 @@ Description:
   application development. Out of the box allows to serve requests using
   'Rserve' package, but flexible enough to integrate with other HTTP servers
   such as 'httpuv'.
-Version: 0.4.1
+Version: 0.4.1001
```

Comment:
Not sure how you handle versioning. Thought a version bump would be helpful. Feel free to change.